hnn_core/dipole.py

```diff
@@ -151,6 +151,8 @@ def average_dipoles(dpls):
         )
     avg_data = np.c_[avg_data].T
     avg_dpl = Dipole(dpls[0].times, avg_data)
+    # The averaged scale should equal all scals in the input dpl list.
+    avg_dpl.scale_applied = dpls[0].scale_applied
```

Comment:
We should also throw a `ValueError` error if the `scale_applied` is not the same for all `dpls`

Reply:
agreed

Reply:
It seems doing so requires changes in `read_dipole` as it ignores the `scale_applied` and only read data from the npy file. That causes the https://github.com/jonescompneurolab/hnn-core/blob/52620b2f05d8ba6c1192a80cac64e2deb0591700/hnn_core/tests/test_dipole.py#L53

to fail the test. My suggestion is: for now we change the `scale_applied` manually for the above test to let it pass the test. But in the future we may include `scale_applied` in the reading/writing of dipole.